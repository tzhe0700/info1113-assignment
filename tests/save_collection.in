ADD COLLECTION spellbooks.csv
SAVE COLLECTION saved_books.csv
EXIT

