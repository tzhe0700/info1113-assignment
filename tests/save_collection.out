user: 1 spellbooks successfully added.

user: Success.

user: Ending Archive process.

